pylint/reporters/ureports/nodes.py

```diff
@@ -117,17 +114,23 @@ class Section(BaseLayout):
     as a first paragraph
     """
 
-    def __init__(self, title=None, description=None, **kwargs):
+    def __init__(
+        self,
+        title: Optional[str] = None,
+        description: Optional[str] = None,
+        **kwargs: Any,
```

Comment:
Can't we type these based on the fact that `BaseLayout` only accepts `children` which is already typed?

Reply:
Sure, I don't understand why this part was over complicated so much.

Reply:
That seems to be a recurring question while working on typing of the codebase 😄